mne/viz/raw.py

```diff
@@ -510,6 +510,7 @@ def _prepare_mne_browse_raw(params, title, bgcolor, color, bad_color, inds,
     """Helper for setting up the mne_browse_raw window."""
     import matplotlib.pyplot as plt
     import matplotlib as mpl
+    plt.ion()
```

Comment:
When a user says `block=True` in their params, the behavior they expect to see from plotting functions is blocking behavior (i.e., non-interactive mode). So doing `ion()` in those cases violates expectation / doesn't follow standard behaviors.

In my mind there needs to be a really good reason to do this. However, the warning you describe sounds to me more like a bug with your particular setup that should be fixed either by upgrading `matplotlib` or `pyqt` or so.